src/main/java/cat/udl/eps/softarch/myroutes/repository/RouteFollowedRepository.java

```diff
@@ -11,7 +11,7 @@
 
 public interface RouteFollowedRepository extends CrudRepository<RouteFollowed, Long>, PagingAndSortingRepository<RouteFollowed, Long> {
     List<RouteFollowed> findByIdContaining(@Param("long") Long id);
-    List<Route> findByCreatedBy(@Param("creator") User creator);
-    List<User> findByRouteOrigin(@Param("origin") Route origin);
-    List<RouteFollowed> findByCreatedByAndRouteOrigin(@Param("creator") User creator, @Param("origin") Route origin);
+    List<RouteFollowed> findByCreatedBy(@Param("creator") User creator);
+    List<User> findByFollows(@Param("follows") Route follows);
```

Comment:
In this case, the function would also return `List<RouteFollowed>`, not `List<User>`